contracts/Vault.vy

```diff
@@ -110,6 +109,7 @@ event StrategyReported:
     strategy: indexed(address)
     gain: uint256
     loss: uint256
+    debtPaid: uint256
     totalGain: uint256
     totalLoss: uint256
     totalDebt: uint256
```

Comment:
This is the only thing we might consider doing v0.3.1 for

Reply:
agree